site/layouts/pricing/list.html

```diff
@@ -13,157 +13,138 @@ <h1 class="display-3">Flexible pricing</h1>
       <!-- CARDS --->
 
       <div class="pricing__plans">
-
-        <!-- HOBBY CARD -->
-        <div class="pricing__card pricing__card--small pricing__card--light-gray ">
-          <h3 class="">Hobby</h3>
-          <div class="d-flex flex-column pricing__card--pricing-details">
-            <div class="pricing__fancy-price justify-content-center mt-4">
-              <div class="pricing__fancy-price__dollar-sign pricing__fancy-price__dollar-sign--big">$</div>
-              <div class="pricing__fancy-price__price--big">0</div>
+        <div class="d-lg-flex mb-5">
+          <!-- HOBBY CARD -->
+          <div class="pricing__card pricing__card--small pricing__card--light-gray mr-3">
+            <h3 class="">Hobby</h3>
+            <div class="pricing__card--pricing-details">
+              <div class="pricing__fancy-price justify-content-center">
+                <div class="pricing__fancy-price__dollar-sign--big pricing__fancy-price__dollar-sign">$</div>
+                <div class="pricing__fancy-price__price pricing__fancy-price__price--big">0</div>
+              </div>
+              <div>forever</div>
             </div>
-            <div class="pricing-muted">forever</div>
-          </div>
-
-          <div class="pricing__btn__wrapper">
-            <a class="btn btn-secondary pricing__btn__secondary pricing__btn mb-3" href="https://app.checklyhq.com/signup" rel="noopener" data-test="pricing-free-signup">Start for free</a>
-            <small class="text-muted text-left align-self-start ml-3 mb-2">* Hobby plan comes with access to Team plan features for 14 days. No credit card required.</small>
-          </div> 
-          
-          <div class="pricing__list pricing__list--hobby">
-            <ul class="pt-3">
-              <li>
-                <i class="ion-ios-checkmark text-secondary mr-2"></i>
-                10 API & 5 Browser checks
-              </li>
-              <li>
-                <i class="ion-ios-checkmark text-secondary mr-2"></i>
-                30 days of aggregated data retention
-              </li>
-              <li>
-                <i class="ion-ios-checkmark text-secondary mr-2"></i>
-                Single-user account
-              </li>
-              <li>
-                <i class="ion-ios-checkmark text-secondary mr-2"></i>
-                Email, Slack, Webhook alerts
-              </li>
-            </ul>
+            <div class="pricing__btn__wrapper">
+              <a class="btn btn-secondary pricing__btn__secondary pricing__btn mb-3" href="https://app.checklyhq.com/signup" rel="noopener" data-test="pricing-free-signup">Start for free</a>
+            </div> 
+            <small class="text-muted text-left align-self-start ml-3 mb-3">* Hobby plan comes with access to Team plan features for 14 days. No credit card required – jump in!</small>
```

Comment:
What do you think of separating the 2 sentences? I feel that might make it a bit more readable.
![image](https://user-images.githubusercontent.com/25155364/216113200-880df222-d4ee-4885-87e5-09e15cb531a3.png)

Though maybe they have to be on the same line because of the `*` 🤔

Reply:
True, but maybe as you said they have to be on the same line because of the *, honestly I'm not sure. Probably we need some input from design team on both reviews.

Reply:
The `*` apparenlty doesn't actually point to anything :sweat_smile: 

I think Alex or someone like that pointed that out recently on Slack

Reply:
Ah yes, here it is: https://checklyhq.slack.com/archives/C012YSC025Q/p1675158535372119